components/vision.py

```diff
@@ -90,13 +90,16 @@ def _get_pose_delta(self, t):
 
     def ping(self):
         """Send a ping to the RasPi to determine the connection latency."""
-        self.ping_time = time.monotonic()
+        t = time.monotonic()
+        if t - self.ping_time > 0.5:
+            # Only ping every 500ms - don't keep smashing NetworkTables
```

Comment:
Given that we're still constantly flushing NetworkTables here, the impact this will have on traffic is negligible.

The performance impact should also be close-to-negligible iff we have the latest wpilib.